recipes/newrelic/infrastructure/ohi/postgres/windows.yml

```diff
@@ -0,0 +1,67 @@
+name: postgres-open-source-integration
+displayName: PostgreSQL Open Source Integration
+description: New Relic install recipe for default Postgres Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-postgresql
+
+dependencies:
+  - infrastructure-agent-installer
+
+installTargets:
+  - type: host
+    os: windows
+    kernelArch: x86_64
+
+# keyword convention for dealing with search terms that could land someone on this instrumentation project
+keywords:
+  - Infrastructure
+  - Integration
+  - postgres
+
+# CLI runs process detection; this is used to filter recipes that are appropriate for matched processes
+processMatch:
+  - postgres\.exe
+
+preInstall:
+  requireAtDiscovery: |
+    powershell -command '
+    try{
+      if (Get-Service "postgresql*" -ErrorAction SilentlyContinue) {
+          echo "1234"
```

Comment:
@NRhzhao should we remove this echo, or update the text?

Reply:
Removed